ranger/data/scope.sh

```diff
@@ -44,7 +44,8 @@ HIGHLIGHT_TABWIDTH=${HIGHLIGHT_TABWIDTH:-8}
 HIGHLIGHT_STYLE=${HIGHLIGHT_STYLE:-pablo}
 HIGHLIGHT_OPTIONS="--replace-tabs=${HIGHLIGHT_TABWIDTH} --style=${HIGHLIGHT_STYLE} ${HIGHLIGHT_OPTIONS:-}"
 PYGMENTIZE_STYLE=${PYGMENTIZE_STYLE:-autumn}
-
+RNGR_OPENSCAD_IMGSIZE=${RNGR_OPENSCAD_IMGSIZE:-1000,1000}
+RNGR_OPENSCAD_COLORSCHEME=${RNGR_OPENSCAD_COLORSCHEME:-Tomorrow Night}	#("Cornfield" "Metallic" "Sunset" "Starnight" "BeforeDawn" "Nature" "DeepOcean" "Solarized" "Tomorrow" "Tomorrow 2" "Tomorrow Night" "Monotone")
```

Comment:
I understand wanting to document the options but we decided against this last time. [See here how we did it instead](https://github.com/ranger/ranger/blame/master/doc/ranger.pod#L1820).

Reply:
removed